lib/includes/template-functions.php

```diff
@@ -1787,46 +1805,49 @@ function epl_listing_toolbar_items() {
  * Retrieves the switch and sorting options normally right aligned
  *
  * @since      3.3
+ * 
+ * @since      3.5 gets the shortcode attributes as param 
  *
  * @param array $args   The arguments.
  *
  * @return string
  */
-function get_epl_listing_toolbar_items( $args = array() ) {
+function get_epl_listing_toolbar_items( $attributes = [], $args = [] ) {
```

Comment:
Not sure about this. IE isn't this better to maintain backward compat?

EG previous was
function get_epl_listing_toolbar_items( $args = array() ) {

Do so shouldn't new options be added AFTER the existing option?

EG
function epl_listing_toolbar_items( $args = [], $attributes = []  ) {

You have
function get_epl_listing_toolbar_items( $attributes = [], $args = [] ) {